README.md

```diff
@@ -290,6 +301,10 @@ f(1.2e6); // "1.2M"
 f(1.3e6); // "1.3M"
 ```
 
+<a name="currencyPrecisionPrefix" href="#currencyPrecisionPrefix">#</a> d3.<b>currencyPrecisionPrefix</b>(<i>step</i>, <i>value</i>) [<>](https://github.com/d3/d3-format/blob/master/src/currencyPrecisionPrefix.js "Source")
```

Comment:
Are these necessarily related to currency? IMO this formatter is useful for formatting large numbers in general, not specific to dealing currency. Although, that use case gives some concrete grounding for decisions, which is great. Probably it's fine to call it `currencyPrecisionPrefix`, but it's actually a more general formatter.